### DISCUSSION TITLE
Query: how does Argo CD reconcile or sync changes if cluster config is manually edited

### DISCUSSION THREAD

**Discussion by renuy** [argoproj/argo-cd#8094] [Q&A]
If app configuration is changed in the cluster manually, does argo cd bring it back to desired state. Are there any knobs to control the same?

**Comment by alexmt** [comment]
hello @renuy ,

Yes, this is supported. You would have to enable auto-syncing and self-healing: 

https://argo-cd.readthedocs.io/en/stable/user-guide/auto_sync/#automatic-self-healing
